src/test/java/com/example/javawebcourse/api/featuretoggle/FeatureToggleExtension.java

```diff
@@ -0,0 +1,69 @@
+package com.example.javawebcourse.api.featuretoggle;
+
+import com.example.javawebcourse.api.annotation.DisabledFeatureToggle;
+import com.example.javawebcourse.api.annotation.EnabledFeatureToggle;
+import com.example.javawebcourse.featuretoggle.FeatureToggleService;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.springframework.core.env.Environment;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+public class FeatureToggleExtension implements BeforeEachCallback, AfterEachCallback {
```

Comment:
Really good. May recommend using a var keyword for local variable types to avoid unnecessary boilerplate